### ISSUE TITLE
loadFont issues

### ISSUE THREAD

**Issue by calvinturbo** [metafloor/bwip-js#214]
Hi, I am trying to use Verdana as font for my barcode, but it looks really weird.

![image](https://user-images.githubusercontent.com/11176065/122032534-05417680-cdd0-11eb-8bcd-25df709d7cc9.png)

My code:

```
   async barcodeGenerator(
      text,
      includeText = true,
      scale = 3,
      height = 10,
      textAlign = 'center',
      barcodeType = 'code128',
    ) {
    const Buffer = require('buffer/').Buffer;


    const fontArrayBuffer = await axios.get('https://g.sendnow.eu/Verdana.otf', {
      responseType: 'arraybuffer',
    });

    const font = Buffer.from(fontArrayBuffer.data, 'binary');

    bwipjs.loadFont('Verdana', 100, 100, font);

    /** Node JS or browser*/
    if (typeof process !== 'undefined' && typeof process.versions.node !== 'undefined') {
      /** Node JS*/

      return await bwipjs
        .toBuffer({
          bcid: barcodeType, // Barcode type
          text: text, // Text to encode
          scale: scale, // 3x scaling factor
          height: height, // Bar height, in millimeters
          includetext: includeText, // Show human-readable text
          textxalign: textAlign, // Always good to set this
          textfont: 'Verdana',
        })
        .toString('base64')
        .catch((err) => {
          return err;
        });
    } else {
      /** browser */
      const canvas = document.createElement('canvas');

      try {
        bwipjs.toCanvas(canvas, {
          bcid: barcodeType, // Barcode type
          text: text, // Text to encode
          scale: scale, // 3x scaling factor
          height: height, // Bar height, in millimeters
          includetext: includeText, // Show human-readable text
          textxalign: textAlign, // Always good to set this
          textfont: 'Verdana',
        });
        return canvas.toDataURL('image/png');
      } catch (e) {
        // `e` may be a string or Error object
      }
    }
  }  
```

**Comment by metafloor** [comment]
I am able to reproduce the issue, but it will take some time and effort to debug (the code in question is based on stb_truetype.h).  For the time being, I recommend using a different font (for example, my local windows copy of verdana.ttf works fine).

**Comment by metafloor** [comment]
The bug turned out to be a type-casting issue in the conversion from C to JavaScript.  The C code returned a 32-bit unsigned integer which was then cast to 16-bit signed integer, then cast to a float.  The JavaScript code had the cast to 16-bit signed integer wrong.  Fixed in v3.0.2.
